script.js

```diff
@@ -0,0 +1,328 @@
+  [3, 3, 'Digit3', '#', '№'], [4, 4, 'Digit4', '$', ';'], [5, 5, 'Digit5', '%', '%'], [6, 6, 'Digit6', '^', ':'],
+  [7, 7, 'Digit7', '&', '?'], [8, 8, 'Digit8', '*', '*'], [9, 9, 'Digit9', '(', '('], [0, 0, 'Digit0', ')', ')'],
+  ['-', '-', 'Minus', '_', '_'], ['=', '=', 'Equal', '+', '+'], ['&#8592 Back', '&#8592 Back', 'Backspace'],
+  ['Tab', 'Tab', 'Tab'], ['q', 'й', 'KeyQ'], ['w', 'ц', 'KeyW'], ['e', 'у', 'KeyE'], ['r', 'к', 'KeyR'],
+  ['t', 'е', 'KeyT'], ['y', 'н', 'KeyY'], ['u', 'г', 'KeyU'], ['i', 'ш', 'KeyI'], ['o', 'щ', 'KeyO'],
+  ['p', 'з', 'KeyP'], ['[', 'х', 'BracketLeft', '{', 'Х'], [']', 'ъ', 'BracketRight', '', '}', 'Ъ'],
+  ['Del', 'Del', 'Delete'], ['CapsLock', 'CapsLock', 'CapsLock'], ['a', 'ф', 'KeyA'], ['s', 'ы', 'KeyS'],
+  ['d', 'в', 'KeyD'], ['f', 'а', 'KeyF'], ['g', 'п', 'KeyG'], ['h', 'р', 'KeyH'], ['j', 'о', 'KeyJ'],
+  ['k', 'л', 'KeyK'], ['l', 'д', 'KeyL'], [';', 'ж', 'Semicolon', ':', 'Ж'], ['&#8242;', 'э', 'Quote', '"', 'Э'],
+  ['\\', '\\', 'Backslash', '|', '/'], ['Enter', 'Enter', 'Enter'], ['Shift', 'Shift', 'ShiftLeft'], ['z', 'я', 'KeyZ'],
+  ['x', 'ч', 'KeyX'], ['c', 'с', 'KeyC'], ['v', 'м', 'KeyV'], ['b', 'и', 'KeyB'], ['n', 'т', 'KeyN'],
+  ['m', 'ь', 'KeyM'], [',', 'б', 'Comma', '<', 'Б'], ['.', 'ю', 'Period', '>', 'Ю'], ['/', '.', 'Slash', '?', ','],
+  ['&#8593', '&#8593', 'ArrowUp'], ['Shift', 'Shift', 'ShiftRight'], ['Ctrl', 'Ctrl', 'ControlLeft'],
+  ['Win', 'Win', 'MetaLeft'], ['Alt', 'Alt', 'AltLeft'], ['Space', 'Space', 'Space'], ['Alt', 'Alt', 'AltRight'],
+  ['&#8592', '&#8592', 'ArrowLeft'], ['&#8595', '&#8595', 'ArrowDown'], ['&#8594', '&#8594', 'ArrowRight'],
+  ['Ctrl', 'Ctrl', 'ControlRight'],
+];
+
+const main = document.createElement('main');
+main.id = ('main');
+document.body.prepend(main);
+
+const textarea = document.createElement('textarea');
+main.append(textarea);
+textarea.id = ('textarea');
+textarea.autofocus = true;
+
+const keyboard = document.createElement('div');
+keyboard.id = ('keyboard');
+main.append(keyboard);
+
+localStorage.setItem('first', '0');
+
+
+// collect the virtual keyboard and add the third element to the array id array for each element
+function keyboardBuilder(allArray) {
+  for (let i = 0; i < allArray.length; i += 1) {
+    const div = document.createElement('section');
+    keyboard.append(div);
+    div.id = array[i][2];
+    document.getElementById(div.id).innerHTML = array[i][localStorage.getItem('second')
+    || localStorage.getItem('first')];
+  }
+}
+
+
+//  highlight keys and value outputcd
+document.addEventListener('keydown', (event) => {
+  for (let i = 0; i < array.length; i += 1) {
+    if (event.code === array[i][2]) {
+      document.querySelector(`#${array[i][2]}`).classList.add('change');
+      if (i !== 60 && i !== 61 && i !== 62 && i !== 53) {
+        event.preventDefault();
+      }
+      if (i !== 13 && i !== 14 && i !== 27 && i !== 28 && i !== 41 && i !== 42 && i !== 53 && i !== 54
```

Comment:
1. Makes sense to extract seperate variable like const specialKeys = [13, 14, 27, 28...]
2. if (!specialKeys.includes(i))